src/include/ipc/topology.h

```diff
@@ -47,6 +47,7 @@ enum sof_comp_type {
 	SOF_COMP_ASRC,		/**< Asynchronous sample rate converter */
 	SOF_COMP_DCBLOCK,
 	SOF_COMP_SMART_AMP,		/**< smart amplifier component */
+	SOF_COMP_CROSSOVER,
```

Comment:
SOF_COMP_PROCESS should be used here. @keyonjie does your UUID PRs ass this type ? @plbossart fyi

Reply:
SOF_COMP_PROCESS doesn't seem to exist in tip-of-tree code. Maybe it is still un-merged?

Reply:
No need to add a new comp type enum, just keep this part intact should work to me. @johnylin76

Reply:
@johnylin76 can you delete this change. I think we are good to merge after this.

Reply:
Hi @lgirdwood , my only concern is that testbench doesn't seem to have uuid token parsing now, which I think should be ported from Linux topology driver.
(mentioned in https://thesofproject.github.io/latest/developer_guides/uuid/index.html#linux-topology-driver)

Then it will cause unsupported comp type error while running testbench for crossover. Shouldn't we port the uuid token parsing to testbench first?

Reply:
> Hi @lgirdwood , my only concern is that testbench doesn't seem to have uuid token parsing now, which I think should be ported from Linux topology driver.
> (mentioned in https://thesofproject.github.io/latest/developer_guides/uuid/index.html#linux-topology-driver)
> 
> Then it will cause unsupported comp type error while running testbench for crossover. Shouldn't we port the uuid token parsing to testbench first?

@johnylin76 @cujomalainey if nobody is using testbench for crossover ATM, it's fine to remove the adding of SOF_COMP_CROSSOVER and leave the tplg_parser gap as TODO?